persistence/jpa/applib/src/main/java/org/apache/isis/persistence/jpa/applib/integration/IsisEntityListener.java

```diff
@@ -66,23 +63,23 @@
 public class IsisEntityListener {
 
     // not managed by Spring (directly)
-    @Inject private ServiceInjector serviceInjector;
+    //@Inject private ServiceInjector serviceInjector;
```

Comment:
optimization: we could store a INJECTONPOINTS_RESOLVED flag into the associated statemanager